apps/next/src/components/Header/ConnectedSites.tsx

```diff
@@ -0,0 +1,249 @@
+import {
+  Button,
+  Divider,
+  getHexAlpha,
+  Popover,
+  Stack,
+  Typography,
+  useTheme,
+} from '@avalabs/k2-alpine';
+import { StackRow } from '../StackRow';
+import {
+  MdCheckCircle,
+  MdChevronRight,
+  MdError,
+  MdOutlineRemoveModerator,
+  MdOutlineUnpublished,
+} from 'react-icons/md';
+import { usePermissionContext } from '@core/ui';
+import { getAllAddressesForAccount } from '@core/common';
+import { Account } from '@core/types';
+import { useTranslation } from 'react-i18next';
+import { useState } from 'react';
+import { useCurrentDomain } from '@/pages/Permissions/useCurrentDomain';
+import { NetworkVMType } from '@avalabs/vm-module-types';
+
+type ConnectedListType = {
+  [key: string]: {
+    accounts: {
+      [key: string]: NetworkVMType;
+    };
+  };
+};
+
+const getAccountConnectedSites = ({
+  list,
+  account,
+}: {
+  list: ConnectedListType;
+  account?: Account;
+}) => {
+  if (!account || !list) {
+    return [];
+  }
+  return Object.values(list).filter((listItem: any) =>
+    Object.keys(listItem?.accounts).some((address) =>
+      getAllAddressesForAccount(account).includes(address),
+    ),
+  );
+};
+
+export const ConnectedSites = ({ activeAccount }) => {
+  const domain = useCurrentDomain();
+  const { permissions, revokeAddressPermisson, isDomainConnectedToAccount } =
+    usePermissionContext();
+  const connectedSitesList = getAccountConnectedSites({
+    list: permissions,
+    account: activeAccount,
+  });
+  const theme = useTheme();
+  const { t } = useTranslation();
+  const isConnected =
+    (isDomainConnectedToAccount &&
+      isDomainConnectedToAccount(
+        domain,
+        getAllAddressesForAccount(activeAccount ?? {}),
+      )) ||
+    false;
```

Comment:
Please consider to use modern syntax.
```suggestion
  const isConnected =
      isDomainConnectedToAccount?.(
        domain,
        getAllAddressesForAccount(activeAccount ?? {}),
      ) ??
    false;
```